lib/system/seqs_v2.nim

```diff
@@ -93,7 +104,13 @@ proc grow*[T](x: var seq[T]; newLen: Natural; value: T) =
     xu.p = cast[typeof(xu.p)](prepareSeqAdd(oldLen, xu.p, newLen - oldLen, sizeof(T), alignof(T)))
   xu.len = newLen
   for i in oldLen .. newLen-1:
-    xu.p.data[i] = value
+    when supportsCopyMem(T):
```

Comment:
If you're particularly worried about performance, we could guard this with a `when not defined(danger):` so it runs faster in `danger` builds?

Reply:
My personal preference is that `setLen` always does the same thing, regardless of the current build mode. While the regression does frustrate me a bit, I don't think it's too much of a problem, and the planned improvements to the backend should offset the regression again.